### ISSUE TITLE
BinaryNet for general Imagenet-type dataset

### ISSUE THREAD

**Issue by Benuraj** [MatthieuCourbariaux/BinaryNet#4]
can we expect this for general Imagenet type of data?

**Comment by MatthieuCourbariaux** [comment]
Yes, we do intend to train a BinaryNet on Imagenet by the end of June.

Besides, a very [recent paper](http://arxiv.org/abs/1603.05279) already tried to train some "XNOR-Nets" on Imagenet.

**Comment by Benuraj** [comment]
Thanks Ill check out the paper.
